### PR TITLE
lulo aggregator adapter

### PR DESCRIPTION
**Pull request by 0xBraunGuy**

### CODE REVIEW

**Comment by llamatester** [comment]
The lulo adapter exports:
        
 
 
    > adapters@1.0.0 test
    > ts-node --transpile-only cli/testAdapter.ts aggregators lulo
    
    🦙 Running LULO adapter 🦙
    ---------------------------------------------------
    Start Date:	Tue, 13 Aug 2024 00:00:00 GMT
    End Date:	Wed, 14 Aug 2024 00:00:00 GMT
    ---------------------------------------------------
    
    SyntaxError: Unexpected token < in JSON at position 0
        at JSON.parse (<anonymous>)
        at parseJSONFromBytes (node:internal/deps/undici/undici:5584:19)
        at successSteps (node:internal/deps/undici/undici:5555:27)
        at fullyReadBody (node:internal/deps/undici/undici:1665:9)
        at processTicksAndRejections (node:internal/process/task_queues:95:5)
        at async specConsumeBody (node:internal/deps/undici/undici:5564:7)
        at async fetchLuloTVL (/home/runner/work/dimension-adapters/dimension-adapters/aggregators/lulo/index.ts:27:28)
        at async /home/runner/work/dimension-adapters/dimension-adapters/helpers/customBackfill.ts:17:22
        at async getChainResult (/home/runner/work/dimension-adapters/dimension-adapters/adapters/utils/runAdapter.ts:44:18)
        at async Promise.all (index 0) {
      chain: 'solana'
    }

**Comment by llamatester** [comment]
The lulo adapter exports:
        
 
 
    > adapters@1.0.0 test
    > ts-node --transpile-only cli/testAdapter.ts aggregators lulo
    
    🦙 Running LULO adapter 🦙
    ---------------------------------------------------
    Start Date:	Wed, 14 Aug 2024 00:00:00 GMT
    End Date:	Thu, 15 Aug 2024 00:00:00 GMT
    ---------------------------------------------------
    
    SOLANA 👇
    Backfill start time: 1/1/2024
    Methodology: Volume is calculated by summing the total USD value of deposited funds in Lulo across all tokens
    End timestamp: 1723679999 (2024-08-14T23:59:59.000Z)

File: aggregators/lulo/index.ts [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+      "x-lulo-api-key": String(process.env.LULO_API_KEY || ''),
+    },
+  };
+
+  const response = await fetch(`${endpoint}?timestamp=${timestamp}`, options)
+
+   const entry: TVLEntry = await response.json()
+
+    return {
+        totalVolume: entry.volume,
```
Comment (dtmkeng) [inline aggregators/lulo/index.ts:30]: could you add daily volume ?

**Review by dtmkeng — CHANGES_REQUESTED** [review]

File: aggregators/lulo/index.ts (hunk 1 above)
Comment (0xBraunGuy) [reply aggregators/lulo/index.ts:30]: it's a lending aggregator, would that be today's total deposited (volume) minus yesterdays total deposited?

Comment (dtmkeng) [reply aggregators/lulo/index.ts:30]: > it's a lending aggregator, would that be today's total deposited (volume) minus yesterdays total deposited?
why it not dexs aggregators aggregators is for aggregators

**Pull request closed without merging**